src/main/java/com/auction/retailService/constant/OrderStatusConstant.java

```diff
@@ -0,0 +1,9 @@
+package com.auction.retailService.constant;
+
+public class OrderStatusConstant {
```

Comment:
Name it `OrderStatus` instead of `OrderStatusConstant`. And you should use an `enum` instead of a class with static strings for this kind of cases:

`
public enum OrderStatus {
    ORDERED,
    PACKING,
    SHIPPED
}
`